src/config.rs

```diff
@@ -1552,23 +1577,52 @@ impl DBConfigManger {
     }
 
     fn set_rate_bytes_per_sec(&self, rate_bytes_per_sec: i64) -> Result<(), Box<dyn Error>> {
-        let mut opt = self.db.as_inner().get_db_options();
-        opt.set_rate_bytes_per_sec(rate_bytes_per_sec)?;
-        Ok(())
+        let mut result = Ok(());
+        self.tablet_factory.for_each_opened_tablet(Box::new(
+            |region_id, suffix, db: &RocksEngine| {
+                let mut opt = db.as_inner().get_db_options();
+                let r = opt.set_rate_bytes_per_sec(rate_bytes_per_sec);
+                if r.is_err() {
+                    result = Err(Box::from(r.err().unwrap()));
+                    error!(
+                        "set_rate_bytes_per_sec {} failed on tablet {}_{}. err:{:?}",
+                        rate_bytes_per_sec, region_id, suffix, &result
+                    );
+                }
+            },
+        ));
+        result
     }
 
     fn set_rate_limiter_auto_tuned(
         &self,
         rate_limiter_auto_tuned: bool,
     ) -> Result<(), Box<dyn Error>> {
-        let mut opt = self.db.as_inner().get_db_options();
-        opt.set_auto_tuned(rate_limiter_auto_tuned)?;
-        // double check the new state
-        let new_auto_tuned = opt.get_auto_tuned();
-        if new_auto_tuned.is_none() || new_auto_tuned.unwrap() != rate_limiter_auto_tuned {
-            return Err("fail to set rate_limiter_auto_tuned".into());
-        }
-        Ok(())
+        let mut result = Ok(());
+        self.tablet_factory.for_each_opened_tablet(Box::new(
+            |region_id, suffix, db: &RocksEngine| {
+                let mut opt = db.as_inner().get_db_options();
+                let r = opt.set_auto_tuned(rate_limiter_auto_tuned);
+                if r.is_err() {
+                    result = Err(Box::from(r.err().unwrap()));
```

Comment:
It fails to set the configuration already, seems meaningless to double check later.

Reply:
good point.